src/ng2-smart-table/components/cell/cell-editors/select-advanced-editor/select-advanced-editor.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, Renderer, OnInit, OnDestroy, ViewChild, ElementRef, ViewEncapsulation } from '@angular/core';
+
+import { SelectAdvancedEditorService } from './select-advanced-editor.service';
+import { DefaultEditor } from '../default-editor';
+
+@Component({
+  selector: 'select-advanced-editor',
+  styleUrls: ['./select-advanced-editor.scss'],
+  encapsulation: ViewEncapsulation.None,
+  providers: [SelectAdvancedEditorService],
+  template: `
+    <div #selectContainer></div>
+  `
+})
+export class SelectAdvancedEditorComponent extends DefaultEditor implements OnInit, OnDestroy {
+
+  multiple: boolean;
+  options: Array<any>;
+  name: string;
+  selected: any;
+  disabled: boolean;
+  @ViewChild('selectContainer') selectContainer: ElementRef;
+
+  constructor(
+    protected selectAdvancedService: SelectAdvancedEditorService,
+    protected renderer: Renderer
+  ) {
+    super();
+  }
+
+  ngOnInit(): void {
```

Comment:
You mean that `:void` is not necessary? If yes, is this a general rule for all the methods with no `return`ed data?